image_cropper_for_web/pubspec.yaml

```diff
@@ -0,0 +1,27 @@
+name: image_cropper_for_web
+description: A Flutter plugin for Web supports cropping images
+repository: https://github.com/hnvn/flutter_image_cropper
+version: 0.0.2
+
+environment:
+  sdk: ">=2.16.1 <3.0.0"
+  flutter: ">=1.17.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
+  
+  image_cropper_platform_interface: ^2.0.0-beta.1
+
+  js: ^0.6.3
+
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  flutter_lints: ^1.0.0
+
+flutter:
```

Comment:
@hnvn 

https://github.com/mono0926/flutter_image_cropper/commit/dfa9fee5531083952679aa8e037a857b6c3f2026 is required for this kind of `generated_plugin_registrant.dart` to be generated.


```dart

import 'package:image_cropper_for_web/image_cropper_for_web.dart';

import 'package:flutter_web_plugins/flutter_web_plugins.dart';

// ignore: public_member_api_docs
void registerPlugins(Registrar registrar) {
  ImageCropperPlugin.registerWith(registrar);
}
```

Reply:
So, it should be generated automatically, right?

Reply:
@hnvn 

> So, it should be generated automatically, right?

Yes👌